candies/cli/parsers/standard.py

```diff
@@ -88,6 +88,11 @@ def prefixes(signature: inspect.Signature) -> str:
     def prefix_of(x: inspect.Parameter) -> str:
         prefix = getattr(x.annotation, 'prefix', '-')
 
+        if prefix == None:
```

Comment:
I think we should use `is` for comparing with `None` or `True`/`False`.

Reply:
Fixed!